samples/bluetooth/central_gatt_write/src/gatt_write_common.c

```diff
@@ -46,6 +52,101 @@ static void write_cmd_cb(struct bt_conn *conn, void *user_data)
 		write_len = 0U;
 		write_rate = 0U;
 		cycle_stamp = k_cycle_get_32();
+
+#if defined(CONFIG_BT_USER_PHY_UPDATE)
+		const struct bt_conn_le_phy_param phy_param[] = {
+			{.options = BT_CONN_LE_PHY_OPT_NONE,
+			 .pref_tx_phy = BT_GAP_LE_PHY_1M,
+			 .pref_rx_phy = BT_GAP_LE_PHY_1M},
+			{.options = BT_CONN_LE_PHY_OPT_NONE,
```

Comment:
I'm a bit confused why Compliance doesn't fail on this. I thought our coding style requires a space after `{` and before `}`. I'd have formatted this as:
```c
const struct bt_conn_le_phy_param phy_param[] = {
	{
		.options = BT_CONN_LE_PHY_OPT_NONE,
		.pref_tx_phy = BT_GAP_LE_PHY_1M,
		.pref_rx_phy = BT_GAP_LE_PHY_1M,
	}, {
		.options = BT_CONN_LE_PHY_OPT_NONE,
		...
	}, {
		.options = ...,
		...
	},
};	
```

Reply:
There is many instance of existing code in subsystem, samples and test that have `{.xyx = ` i.e. without a space after `{`. Something must have been natural to me that made me define the declaration this way.

That said, the style you mention is what I personally prefer tho. Hopefully if this is what will be enforced by compliance in the future; I will make the change.